### DISCUSSION TITLE
Ticket ID: 2234536

### DISCUSSION THREAD

**Discussion by ghost** [community/community#60040] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Username: ShanmukhaUV
I have raised a Github Support request. This is related to my account which was suspended for unknown reason even though this is the first time I am logging into my github account. Below is what I could see every time I login with my creds

**Access to your account has been suspended due to a violation of our [Terms of Service](https://docs.github.com/articles/github-terms-of-service).**

**Please [contact support](https://support.github.com/contact/reinstatement) for more information.**

**Comment by 3urobeat** [comment]
I don't think anyone can help you with this problem here, only GitHub support can.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ShannmukhV, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! Please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by Ahmadgadimi** [comment]
Ahmadghademi210@gmali.com
